pkg/license/reporter_test.go

```diff
@@ -21,155 +21,191 @@ import (
 	esv1 "github.com/elastic/cloud-on-k8s/pkg/apis/elasticsearch/v1"
 	kbv1 "github.com/elastic/cloud-on-k8s/pkg/apis/kibana/v1"
 	commonlicense "github.com/elastic/cloud-on-k8s/pkg/controller/common/license"
-	essettings "github.com/elastic/cloud-on-k8s/pkg/controller/elasticsearch/settings"
 	"github.com/elastic/cloud-on-k8s/pkg/controller/kibana"
 	"github.com/elastic/cloud-on-k8s/pkg/utils/k8s"
 )
 
 const operatorNs = "test-system"
 
-func Test_Get(t *testing.T) {
-	es := esv1.Elasticsearch{
-		Spec: esv1.ElasticsearchSpec{
-			NodeSets: []esv1.NodeSet{{
-				Count: 10,
-			}},
-		},
-	}
-	licensingInfo, err := NewResourceReporter(k8s.FakeClient(&es), operatorNs).Get()
-	assert.NoError(t, err)
-	assert.Equal(t, "21.47GB", licensingInfo.TotalManagedMemory)
-	assert.Equal(t, "1", licensingInfo.EnterpriseResourceUnits)
+func TestGet(t *testing.T) {
```

Comment:
I realize both are not directly from this PR, but:

It doesn't look like we have a test exercising the maxERUs branch, correct? It might be worth adding one if possible

Nit: Since they don't share anything, at this point it might be simpler to break each case out into separate tests just to make it easier to run a specific test if you need to

Reply:
There are tests for maxERUs in the licence tests.

I am not sure I understand your comment about individual tests. You can already run specific tests by invoking something like `go test -run='TestGet/elasticsearch_defaults' ./pkg/license/...`

Reply:
You're right, I was being goofy 🤦‍♀️